src/shogun2-init/src/main/java/de/terrestris/shogun2/service/InitializationService.java

```diff
@@ -37,9 +37,6 @@
 	@Qualifier("genericDao")
 	private GenericHibernateDao<PersistentObject, Integer> dao;
 
-	/**
-	 * The password encoder that is used to encode the password of a user.
-	 */
```

Comment:
It's very likely, this has been accidentally removed by a rebase. I'll bring it back shortly...